internal/cli/kraft/lib/create/create.go

```diff
@@ -0,0 +1,315 @@
+		panic(err)
+	}
+
+	return cmd
+}
+
+func (*CreateOptions) Pre(cmd *cobra.Command, _ []string) error {
+	ctx, err := packmanager.WithDefaultUmbrellaManagerInContext(cmd.Context())
+	if err != nil {
+		return err
+	}
+	cmd.SetContext(ctx)
+	return nil
+}
+
+func (opts *CreateOptions) Run(ctx context.Context, args []string) error {
+	var err error
+
+	if len(args) > 0 {
+		opts.ProjectName = args[0]
+	}
+
+	cwd, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+
+	if !config.G[config.KraftKit](ctx).NoPrompt {
+		if !opts.GitInit {
+			input := textinput.New("Do you want to intialise library with git [Y/n]:")
+			input.Placeholder = "Enter Y for yes"
+			inputValue, err := input.RunPrompt()
+			if err != nil {
+				return err
+			}
+			if inputValue == "Y" || inputValue == "y" {
+				opts.GitInit = true
+			}
+		}
+
+		if len(opts.ProjectName) == 0 {
+			input := textinput.New("Project name:")
+			input.Placeholder = "Project name cannot be empty"
+			opts.ProjectName, err = input.RunPrompt()
+			if err != nil {
+				return err
+			}
+		}
+
+		if len(opts.ProjectPath) == 0 {
+			input := textinput.New("Work directory:")
+			input.Placeholder = "Where to create library"
+			input.InitialValue = cwd
+			opts.ProjectPath, err = input.RunPrompt()
+			if err != nil {
+				return err
+			}
+		}
+
+		if !opts.SoftPack {
+			input := textinput.New("Do you want to package it [Y/n]:")
+			input.Placeholder = "Y for yes or n for no"
+			inputValue, err := input.RunPrompt()
+			if err != nil {
+				return err
+			}
+			if inputValue == "Y" || inputValue == "y" {
+				opts.SoftPack = true
+			}
+		}
+
+		if len(opts.LibraryName) == 0 {
+			input := textinput.New("Library name:")
+			input.Placeholder = "Library name cannot be empty "
+			input.InitialValue = "lib-" + opts.ProjectName
+			opts.LibraryName, err = input.RunPrompt()
+			if err != nil {
+				return err
+			}
+		}
+
+		if len(opts.LibraryKName) == 0 {
+			input := textinput.New("Library kname:")
+			input.Placeholder = "Library kname cannot be empty"
+			input.InitialValue = "LIB" + strings.ToUpper(strings.ReplaceAll(opts.ProjectName, "-", ""))
+			opts.LibraryKName, err = input.RunPrompt()
+			if err != nil {
+				return err
+			}
+		}
+
+		// if len(opts.Description) == 0 {
+		// 	input := textinput.New("Description:")
+		// 	input.Placeholder = "Description"
+		// 	input.InitialValue = ""
+		// 	opts.Description, err = input.RunPrompt()
+		// 	if err != nil {
+		// 		return err
+		// 	}
+		// }
+
+		if len(opts.Version) == 0 {
+			input := textinput.New("Version:")
+			input.Placeholder = "Version"
+			input.InitialValue = "1.0.0"
+			opts.Version, err = input.RunPrompt()
+			if err != nil {
+				return err
+			}
+		}
+
+		if len(opts.AuthorName) == 0 {
+			input := textinput.New("Author name:")
+			input.Placeholder = "Author name cannot be empty"
+			input.InitialValue = os.Getenv("USER")
+			opts.AuthorName, err = input.RunPrompt()
+			if err != nil {
+				return err
+			}
```

Comment:
See comment above

Also, I would not assume that `USER` is correct, better to have no initial value

Reply:
Removed 👍🏽